### PR TITLE
fix: make the component work in swarm-extension

### PR DESCRIPTION
**Pull request by vojtechsimetka**
Somewhat addresses #518 . I am sure better solution could be designed, but with this the Swarm Extension works.

### CODE REVIEW

**Comment by bee-worker** [comment]
## 🐝 PR preview in Swarm

**Preview URL:** https://bah5acgza4wvvxnd2niaif3yl6dung3a3ifwwhz5rrg3dsql6gc4oynunal2q.bzz.link
**Swarm Hash:** `e5ab5bb47a6a0082ef0bf0e8d36c1b416d63e7b189b639417e30b8ec368d02f5`
**Commit Hash:** fe87ccde4ed971f7edb66f4e4e48d70a8a21a65f
**Commit Message:** fix: make the component work in swarm-extension

File: src/config.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
+// FIXME: in the Swarm Extension process is not defined and would throw an error
+// see https://github.com/ethersphere/bee-dashboard/issues/518
+export function getEnvVariable(variableName: string): string | undefined {
+  if (typeof process === 'object') return process.env[variableName]
```
Comment (AuHau) [inline src/config.ts:4]: Did you test this for the normal Dashboard? I remember that the test `typeof process` did not work with the CRA mechanism of the env. replacement. Maybe you have to extend it to `process.env`...

Comment (vojtechsimetka) [reply src/config.ts:4]: You are right. However, the test `process && process.env` complaints in the Swarm Extension so I wonder if removing the env variables here and injecting them instead would not be easier.

**Comment by Cafe137** [comment]
Swarm Extension has been released with v0.20.2

**Pull request closed without merging**